### DISCUSSION TITLE
Please help NoneType object is not iterable" Error When Adding API Key in Dify Cloud

### DISCUSSION THREAD

**Discussion by fabiofurlano** [langgenius/dify#5179] [General]
Description:
I am encountering an error when trying to add a new SerpApi API key in Dify Cloud. The error message displayed is: NoneType object is not iterable.

Steps to Reproduce:

Go to the "Web Searcher" workflow in Dify Cloud.
Click on "Set Up Auth" to add a new API key.
Enter the SerpApi API key.
Click "Save".
Expected Result:
The API key should be saved without any errors.

Actual Result:
An error message appears: NoneType object is not iterable.

Additional Information:

The API key has been verified and works with Postman.
The issue occurs with multiple API keys.
Using Dify Cloud version.
Attached is a screenshot showing the error message.
Screenshots: I will delete api key in pic already...
![Image 14-06-24 at 00 41](https://github.com/langgenius/dify/assets/38840658/61307c33-c61f-4f2c-a08a-1a25e3d8b411)

Thank you for your help.

**Comment by crazywoola** [comment]
Hello we are resolving this right now https://github.com/langgenius/dify/pull/5190

**Reply by crazywoola** [reply]
Should be resolved in main branch, we will deploy it to cloud later.

**Reply by fabiofurlano** [reply]
really great thank this platform is great
